interactions/client/client.py

```diff
@@ -1597,12 +1600,12 @@ async def _dispatch_interaction(self, event: RawGatewayEvent) -> None:
                     elif autocomplete := self._global_autocompletes.get(str(auto_opt.name)):
                         callback = autocomplete
                     else:
-                        breakpoint()
+                        # breakpoint()
                         raise ValueError(f"Autocomplete callback for {str(auto_opt.name)} not found")
 
                     await self.__dispatch_interaction(
                         ctx=ctx,
-                        callback=callback(ctx),
+                        callback=callback(ctx, **ctx.kwargs),
```

Comment:
I would argue passing kwargs to autocompletes is both unexpected for most use cases, and unnecessary; namely because in the unlikely event someone actually needs them, they are available through `ctx.kwargs`
Adding this only adds bloat and a barrier of entry

Reply:
Really? Because this example doesn't work without it: 
https://github.com/interactions-py/interactions.py/blob/5.x/docs/src/Guides/03%20Creating%20Commands.md#i-need-more-than-25-choices

Reply:
Regardless, reverted. This can be addressed in another PR.